### DISCUSSION TITLE
Want backdrop filter for making things glossy 😃

### DISCUSSION THREAD

**Discussion by apneduniya** [fastn-stack/fastn#1344] [Ideas & RFCs]
![image](https://github.com/fastn-stack/fastn/assets/113763427/f19b6861-a4a7-4d3a-9bec-da3b30ef7b7c)

Want to create things like this via fastn :)

**Comment by amitu** [comment]
@apneduniya can you share the corresponding HTML/CSS? Looks like a good feature, we should implement.

**Comment by apneduniya** [comment]
Yeah sure 😄 
This might be a good example: https://codepen.io/codesuey/pen/mdbxmGo

**Reply by amitu** [reply]
`backdrop-filter: blur(4px);` is the magic incantation. Got it. Thanks.

**Comment by apneduniya** [comment]
> `backdrop-filter: blur(4px);` is the magic incantation. Got it. Thanks.

Opacity is also one reason with this. Its my pleasure to share this with you 😄 . Will be waiting for this to come on fastn soon 🙃

**Reply by amitu** [reply]
Just send a PR already.
